rust/src/router/plain_transport.rs

```diff
@@ -111,19 +116,99 @@ pub struct PlainTransportDump {
     pub data_consumer_ids: Vec<DataConsumerId>,
     pub recv_rtp_header_extensions: RecvRtpHeaderExtensions,
     pub rtp_listener: RtpListener,
-    pub max_message_size: usize,
+    pub max_message_size: u32,
     pub sctp_parameters: Option<SctpParameters>,
     pub sctp_state: Option<SctpState>,
     pub sctp_listener: Option<SctpListener>,
     pub trace_event_types: String,
     // PlainTransport specific.
     pub rtcp_mux: bool,
     pub comedia: bool,
-    pub tuple: TransportTuple,
+    pub tuple: Option<TransportTuple>,
```

Comment:
This is optional and may come empty.

Reply:
Why is it optional if it wasn't optional before? Also `PlainTransportStat` has non-optional `tuple` field now for some reason.